### DISCUSSION TITLE
Won't detect audio media file to download

### DISCUSSION THREAD

**Discussion by JasonD71** [aclap-dev/video-downloadhelper#296] [Help and Q&A]
### Your question

Has anyone noticed that the latest version does not recognize/detect audio media files for download from the windows firefox web browser?? Very frustrating!!!!!!! Any news on an update to fix this?

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): a067e3e71db44877
platform: x86-32 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "i686",
    "node": "10"
  },
  "home": "C:\\Users\\Jason",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
Where do you see it happen ? Any examples to share.
